src/include/units/physical/dimensions.h

```diff
@@ -110,6 +110,30 @@ struct dim_surface_tension : derived_dimension<Child, U, exp<F, 1>, exp<L, -1>>
 template<typename Child, Unit U, DimensionOf<dim_force> F, DimensionOf<dim_area> A>
 struct dim_pressure : derived_dimension<Child, U, exp<F, 1>, exp<A, -1>> {};
 
+template <typename Child, Unit U, DimensionOf<dim_voltage> V, DimensionOf<dim_time> T, DimensionOf<dim_length> L>
+struct dim_magnetic_induction : derived_dimension<Child, U, exp<V, 1>, exp<T, 1>, exp<L, -2>> {};
+
+template<typename Child, Unit U, DimensionOf<dim_magnetic_induction> B, DimensionOf<dim_area> A>
+struct dim_magnetic_flux : derived_dimension<Child, U, exp<B, 1>, exp<A, 1>> {};
+
+template<typename Child, Unit U, DimensionOf<dim_magnetic_flux> F, DimensionOf<dim_electric_current> I>
+struct dim_inductance : derived_dimension<Child, U, exp<F, 1>, exp<I, -1>> {};
+
+template<typename Child, Unit U, DimensionOf<dim_resistance> R>
+struct dim_conductance : derived_dimension<Child, U, exp<R, -1>> {};
+
+// template<typename Child, Unit U, DimensionOf<dim_time> T>
+// struct dim_radioactivity : derived_dimension<Child, U, exp<T, -1>> {};
```

Comment:
Yeah, I know this sucks ;-( I have to find out a solution for this Downcasting Facility or we will have to disable it for good.